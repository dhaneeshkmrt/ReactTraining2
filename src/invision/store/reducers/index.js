import { combineReducers } from "redux";

import { movieReducer } from "./movie.reducer";
import { popupReducer } from "./popup.reducer";

export const reducers = combineReducers({
  movies: movieReducer,
  popups: popupReducer
})

