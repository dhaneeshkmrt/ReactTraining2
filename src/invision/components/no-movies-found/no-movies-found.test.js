import TestRenderer from 'react-test-renderer';
import React from 'react';
import { Provider } from 'react-redux';
import { store } from "../../store";
import NoMoviesFound from '../no-movies-found/no-movies-found';

describe('no-movies-found:', () => {
  test('should render', () => {
    const searchBar = TestRenderer.create(<Provider store={store}><NoMoviesFound /></Provider >);
    const json = searchBar.toJSON();
    expect(json).toMatchSnapshot();
  })
});