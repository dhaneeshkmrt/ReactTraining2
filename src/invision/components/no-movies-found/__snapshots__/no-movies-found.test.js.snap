// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no-movies-found: should render 1`] = `
<div
  className="no-movies-found"
>
  <h1>
     No Movies Found
  </h1>
</div>
`;
